File: javascript/raindrops/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (number) => {
  const sound = {
    3: "Pling",
    5: "Plang",
    7: "Plong",
  }
  let result = ""
  for (const i in sound) {
    if (number % Number(i) === 0) {
      result += sound[i]
    }
  }
  return result || number.toString();
};

File: solutions/go/raindrops/1/raindrops.go
package raindrops

import "fmt"

func Convert(number int) string {
	var divisor []int = []int{3, 5, 7}
	sounds := []string{"Pling", "Plang", "Plong"}
	var result string
	for i := 0; i < len(divisor); i++ {
		if number%divisor[i] == 0 {
			result += sounds[i]
		}
	}
	if result == "" {
		return fmt.Sprintf("%d", number)
	}
	return result
}
